internal/controller/cluster/reconciler.go

```diff
@@ -274,9 +274,9 @@ func (r *SingleClusterReconciler) Reconcile() (result ctrl.Result, recErr error)
 		return reconcile.Result{}, err
 	}
 
-	// Try to recover pods only when MaxIgnorablePods is set
-	if r.aeroCluster.Spec.RackConfig.MaxIgnorablePods != nil {
-		if res := r.recoverIgnorablePods(); !res.IsSuccess {
+	// Try to recover pods only when length of ignorablePodNames is greater than zero
```

Comment:
try to add some "why?" in comments. Current comment is exactly saying what like 278 is doing.